tortoise/verifying.go

```diff
@@ -94,27 +144,25 @@ func (v *verifying) sumGoodBallots(logger log.Log, ballots []tortoiseBallot) (we
 	sum := weightFromUint64(0)
 	n := 0
 	for _, ballot := range ballots {
-		if !v.isGood(logger, ballot) {
+		rst := v.isGood(logger, ballot)
+		if rst == bad {
 			continue
 		}
-		v.goodBallots[ballot.id] = struct{}{}
-		sum = sum.add(ballot.weight)
-		n++
+		v.goodBallots[ballot.id] = rst
```

Comment:
use checkIsGood?

Reply:
i will drop checkIsGood i guess, it makes more sense to check that rst is god